Delcam.ProductInterface.PowerMILL/Classes/PMProject.cs

```diff
@@ -451,6 +454,27 @@ public void CreateBlock(PMBoundary boundary, double ZMin, double ZMax)
             _powerMILL.DoCommand(string.Format("EDIT BLOCK ZMIN \"{0}\"", ZMin));
             _powerMILL.DoCommand(string.Format("EDIT BLOCK ZMAX \"{0}\"", ZMax));            
             _powerMILL.DoCommand("BLOCK ACCEPT");
+            BoundingBox boundingBox = GetBlockLimits();
+            return boundingBox;
+        }
+
+        /// <summary>
+        /// Returns the block limits
+        /// </summary>
+        public BoundingBox GetBlockLimits()
+        {
+            string tpBlockXmin = _powerMILL.DoCommandEx("PRINT PAR $Block.Limits.XMin").ToString().Replace("(REAL)", "").Replace("[L]", "").Trim();
```

Comment:
If you use "PRINT PAR TERSE" rather than "PRINT PAR" then you don't need to get rid of the "(REAL)" string.

Reply:
I've always wondered what "TERSE" do